commander/src/bootstrapping/commands/endpoint/list.ts

```diff
@@ -0,0 +1,94 @@
+import { Schema } from '@liskhq/lisk-codec';
+import { Flags as flagParser } from '@oclif/core';
+import { BaseIPCClientCommand } from '../base_ipc_client';
+
+export class ListCommand extends BaseIPCClientCommand {
+	static description = 'Invokes the provided endpoint.';
```

Comment:
I believe this should be `List endpoints`? 🧐

Reply:
🤦🏽‍♂️; thank you, yes, updated.